build.gradle

```diff
@@ -107,3 +107,14 @@ googleJavaFormat {
 testlogger {
   theme 'mocha'
 }
+
+jar {
```

Comment:
Can you look into putting it into the "createScript" function. Ideally we should have two separate .jars (with their own deps included) as clutz and gents don't really share that much code or functionality.

If you can't get that to work, it's ok too, but at least document the current approach is not ideal, so that maybe someone else can improve it later.

Reply:
Will do.  I'll try to see if I can have separate Jar files, one for gents and one for clutz.  Then the Main class in this PR won't be needed.